articles/getting-started-with-jwt-using-angular8-and-nodejs/index.md

```diff
@@ -0,0 +1,362 @@
+JSON Web Token (JWT) is an RFC standard that defines a compact way of securing
+the data transmitted between the parties as a JSON object. The information is
+digitally signed, meaning and it can then be verified and signed. JWTs are
+signed using a secret key or a public/private key pair using RSA/ECDSA
+algorithms. In this guide, I will use a secret key.
+
+## Prerequisites
+
+To accomplish the task using this guide, you will need:
+
+-   Basic knowledge of [Node.js](https://nodejs.dev/learn), [Angular 11](https://www.techiediaries.com/angular-11-crud-rest-api-tutorial/) and [JWT](https://jwt.io/introduction)
+
```

Comment:
Talk of 'Some knowledge of' instead of 'Basic knowledge of'... It would be discouraging if a reader with 'basic knowledge of ' what is mentioned above, finds it hard to understand this tutorial.